### PR TITLE
Loadimage

### PR DESCRIPTION
**Pull request by alinkaKuliesh**
@lukassplitthoff  please check

### CODE REVIEW

File: code/imageloading.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 '''
```
Comment (lukassplitthoff) [inline code/imageloading.py:5]: this line can be removed

File: code/imageloading.py [hunk 2]
```diff
@@ -22,4 +23,8 @@ def load_image(path):
     """
     Im = sio.loadmat(path)
     
-    
\ No newline at end of file
+
+path = '2dpeakfinder/data/brain.mat'
```
Comment (lukassplitthoff) [inline code/imageloading.py]: this is neither a function nor a test, please correct it

File: code/imageloading.py [hunk 3]
```diff
@@ -22,4 +23,8 @@ def load_image(path):
     """
     Im = sio.loadmat(path)
```
Comment (lukassplitthoff) [inline code/imageloading.py:20]: it should return a "pass" to keep the structure

File: code/imageloading.py [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 '''
 import scipy.io as sio
+import matplotlib.pyplot as plt
 
 def load_image(path):
```
Comment (lukassplitthoff) [inline code/imageloading.py:8]: I think the purpose of this function should be to load the matlab file and return a numpy array

a separate function should load this numpy array and display it

**Review by lukassplitthoff — CHANGES_REQUESTED** [review]

**Comment by alinkaKuliesh** [comment]
@lukassplitthoff  can you please check again

**Review by lukassplitthoff — APPROVED** [review]
I approve the merge

**Pull request merged**